TrackPropagation/Geant4e/src/Geant4ePropagator.cc

```diff
@@ -312,18 +310,11 @@ std::pair<TrajectoryStateOnSurface, double> Geant4ePropagator::propagateGeneric(
 
   theG4eManager->InitTrackPropagation();
 
-  // re-initialize navigator to avoid mismatches and/or segfaults
-  theG4eManager->GetErrorPropagationNavigator()->LocateGlobalPointAndSetup(
-      g4InitPos, &g4InitMom, /*pRelativeSearch = */ false, /*ignoreDirection = */ false);
-
   bool continuePropagation = true;
   while (continuePropagation) {
     iterations++;
     LogDebug("Geant4e") << std::endl << "step count " << iterations << " step length " << finalPathLength;
 
-    // re-initialize navigator to avoid mismatches and/or segfaults
-    theG4eManager->GetErrorPropagationNavigator()->LocateGlobalPointWithinVolume(g4eTrajState.GetPosition());
```

Comment:
this should not be removed.

Reply:
It was mistake to make modifications on top of 12_4_12, now it is done on top of master.